telemetry/telemetry.go

```diff
@@ -0,0 +1,111 @@
+// As part of the White Glove White Glove project,
+// this is a telemetry package to support posting summary and cancellation events
+// to a given webhook endpoint with a reference project id in the shape of an event grid
+// custom event.
+
+package telemetry
+
+import (
+	"bytes"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-storage-azcopy/common"
+)
+
+var (
+	// ReportingAPIURI the Webhook URI
+	ReportingAPIURI string
+	// ProjectReferenceID the project reference ID
+	ProjectReferenceID string
+)
+
+type telemetry struct {
+	JobID     string      `json:"jobID"`
```

Comment:
Minor: Out of curiosity, what's the reason why this is two types, instead of one?  Just asking because I would have expected the projectId to be at the top level of the final Json, although it looks like Telemetry ends up nested in reportEvent.